test/metric_value_benchmark/mem_test.go

```diff
@@ -10,41 +10,49 @@ import (
 	"time"
 
 	"github.com/aws/amazon-cloudwatch-agent-test/test/metric"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric/dimension"
+
 	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
 )
 
 type MemTestRunner struct {
-	BaseTestRunner
+	test_runner.BaseTestRunner
+	Base test_runner.BaseTestRunner
```

Comment:
Why does the struct need two copies of the BaseTestRunner?

Reply:
apparently that's not how it works. if I wanna reference member variables from Base then I need to specify Base, but if I wanna just use the methods defined for base then I cannot specify Base. I guess in that case I will have to do Base.call()

Reply:
🙁 well, in that case, I would rather drop the first one and then update usage everywhere to do `Base.function()`. I don't like that, but it's better than specifying it twice. 

Or, do you need member variables from base or can you expose it via a function?

Reply:
Im still a little confused why we need both here... was it a compilation error or a runtime error? Because I atleast tried `t.DimensionFactory.GetDimensions` instead of `t.Base.DimensionFactory.GetDimensions` and it didnt complain in my IDE.

Reply:
wow that didn't work when I tried so that's why I did this Base thing. dang. It did complain when I did go test but somehow not anymore. I must have misinterpreted something